### PR TITLE
Feature/cdctoolbox 386 2

### PR DESCRIPTION
**Pull request by bmmsantos**

### CODE REVIEW

File: src/services/copyConfig/policies/policyOptions.js [hunk 1]
```diff
@@ -31,158 +31,188 @@ class PolicyOptions extends Options {
...
+        name: PolicyOptions.#emailVerification,
+        propertyName: PolicyOptions.#emailVerification,
+        value: true,
+        tooltip: 'POLICIES_EMAIL_VERIFICATION',
+      }
+      if (response.emailVerification.nextURL) {
+        evOption['branches'] = [
+          {
+            id: `${evOption.id}-Link`,
+            name: 'Include Customize Redirection URL',
```
Comment (luismarques-io) [inline src/services/copyConfig/policies/policyOptions.js:93]: Can we improve this in a way that it doesn't require to have this text directly here? Maybe storing it in "PolicyOptions."?

File: src/services/copyConfig/policies/policyOptions.js [hunk 2]
```diff
@@ -31,158 +31,188 @@ class PolicyOptions extends Options {
...
+        name: PolicyOptions.#doubleOptIn,
+        propertyName: PolicyOptions.#doubleOptIn,
+        value: true,
+        tooltip: 'POLICIES_DOUBLE_OPT_IN',
+      }
+      doiOption.branches = []
+      if (response.doubleOptIn.nextURL) {
+        doiOption.branches.push({
+          id: `${id}-nextUrl-Link`,
+          name: 'Include Customize Redirection URL',
```
Comment (luismarques-io) [inline src/services/copyConfig/policies/policyOptions.js:188]: Can we improve this in a way that it doesn't require to have this text directly here? Maybe storing it in "PolicyOptions."?

File: src/services/copyConfig/policies/policyOptions.js [hunk 3]
```diff
@@ -31,158 +31,188 @@ class PolicyOptions extends Options {
...
+          link: `${PolicyOptions.#doubleOptIn}.nextURL`,
+          value: true,
+        })
+      }
+      if (response.doubleOptIn.nextExpiredURL) {
+        doiOption.branches.push({
+          id: `${id}-nextExpiredUrl-Link`,
+          name: 'Include Customize Expired Redirection URL',
```
Comment (luismarques-io) [inline src/services/copyConfig/policies/policyOptions.js:196]: Can we improve this in a way that it doesn't require to have this text directly here? Maybe storing it in "PolicyOptions."?

**Review by luismarques-io — CHANGES_REQUESTED** [review]

File: src/inject/main.css
[the remark this answers is not in the document]
Comment (bmmsantos) [reply src/inject/main.css:1]: done.

File: src/services/copyConfig/emails/emailOptions.js [hunk 4]
```diff
@@ -51,15 +50,15 @@ class EmailOptions extends Options {
         value: true,
       })
     }
-    if (response.emailNotifications.welcomeEmailTemplates) {
+    if (response.emailNotifications && response.emailNotifications.welcomeEmailTemplates) {
```
[the remark this answers is not in the document]
Comment (bmmsantos) [reply src/services/copyConfig/emails/emailOptions.js]: improved

File: src/services/copyConfig/emails/emailOptions.js [hunk 5]
```diff
@@ -51,15 +50,15 @@ class EmailOptions extends Options {
...
       const id = 'welcomeEmailTemplates'
       this.options.branches.push({
         id: id,
         name: emailTemplateNameTranslator.translateInternalName(id),
         value: true,
       })
     }
-    if (response.emailNotifications.accountDeletedEmailTemplates) {
+    if (response.emailNotifications && response.emailNotifications.accountDeletedEmailTemplates) {
```
[the remark this answers is not in the document]
Comment (bmmsantos) [reply src/services/copyConfig/emails/emailOptions.js]: improved

File: src/services/copyConfig/emails/emailOptions.js [hunk 6]
```diff
@@ -107,7 +128,7 @@ class EmailOptions extends Options {
         value: true,
       })
     }
-    if (response.emailNotifications.confirmationEmailTemplates) {
+    if (response.emailNotifications && response.emailNotifications.confirmationEmailTemplates) {
```
[the remark this answers is not in the document]
Comment (bmmsantos) [reply src/services/copyConfig/emails/emailOptions.js]: improved

File: src/services/copyConfig/policies/policyOptions.js (hunk 1 above)
Comment (bmmsantos) [reply src/services/copyConfig/policies/policyOptions.js]: The text was added to the translations file. We are now using translation on this node.

File: src/services/copyConfig/policies/policyOptions.js (hunk 2 above)
Comment (bmmsantos) [reply src/services/copyConfig/policies/policyOptions.js]: The text was added to the translations file. We are now using translation on this node.
The same was added on email options.

File: src/services/copyConfig/policies/policyOptions.js (hunk 3 above)
Comment (bmmsantos) [reply src/services/copyConfig/policies/policyOptions.js]: The text was added to the translations file. We are now using translation on this node.
The same was added on email options.

**Review by luismarques-io — APPROVED** [review]

**Pull request merged**